scripts/sign_encrypt.py

```diff
@@ -129,6 +138,7 @@ def get_args(logger):
 
 def main():
     from Cryptodome.Signature import pss
+    from Cryptodome.Signature import PKCS1_v1_5
```

Comment:
I think this should be `pkcs1_15` instead of `PKCS1_v1_5`.

Reply:
I think both `PKCS1_v1_5` and `pkcs1_15` should work. See [here](https://github.com/Legrandin/pycryptodome/blob/master/lib/Crypto/Signature/PKCS1_v1_5.py#L39).

Reply:
@wamserma is right IMO. `PKCS1_v1_5.verify` will return `false` instead of raising `ValueError`, that would be a problem since you're explicitely catching `ValueError` below. Also `pkcs1_15` is the new API so better use it.